### PR TITLE
Present criteria as not-tables

### LINKED ISSUE #99: Proposal: Human-readable format should not be a spreadsheet or table-style document
As the title suggests, I propose that the human readable format for this baseline stuff should be something *not* a table or spreadsheet. The long-form text of exposition and explanation should not be typeset into a skinny spreadsheet-like structure. For a visual example, this project's current website looks like this on my phone:

image

Too much whitespace is dedicated to the data columns with the least amount of text.  Even when I scroll across, the table permits at most 3 words per line. It's not great ;) Yes, it looks different on my computer with a bigger screen and different layout provided by the browser on this table.

The longer "criteria details" area below the table feels informative enough without needing the table above it.

Thoughts?

### PR DESCRIPTION
**Pull request by funnelfiasco**
This improves small-screen readability and removes/moves extraneous information for easier skimming.

Some choices I made:

* I went with just the ID and criterion because the level is in the heading and the category is in the ID. I added them to the detailed view for completeness sake.
* I could have iterated over the three levels instead of copy pasta, but
    1. The more code blocks we add, the more complicated the template gets
    2. I think we'll eventually want to add a brief description of each level before we start listing the criteria, so this is future-proofing against that a bit
* Edit to add the other thing I was going to put here: this approach requires no muddling with CSS, adding plugins, etc and still achieves the overarching goal

Preview available as https://funnelfiasco.github.io/security-baseline/

Fixes #99

### CODE REVIEW

**Comment by eddie-knight** [comment]
Thanks for making this happen, I think this is the right approach.

Something feels just a bit off with the list, but I can't put my finger on it. Perhaps it's the lack of a leading list mark on each line, or the use of an emdash 🤔 Just feels like it's _almost_ there, but not quite.

**Comment by jordansissel** [comment]
This is much improved when I try to read this version compared to the table (especially on mobile)

File: cmd/template.md [hunk 1]
```diff
@@ -15,11 +15,32 @@ For more information on the project and to make contributions, visit the [GitHub
...
-| --- | -------------- | -------- | -------- |
+* [Level 1](#level-1)
+* [Level 2](#level-2)
+* [Level 3](#level-3)
+
+### Level 1
+
+{{- range .Criteria }}
+{{if eq .MaturityLevel 1}}
+[{{ .ID }}]({{ .ID | asLink }}) — {{ .CriteriaText | addLinks }}
```
Comment (eddie-knight) [inline cmd/template.md:26]:
```suggestion
**[{{ .ID }}]({{ .ID | asLink }})** - {{ .CriteriaText | addLinks }}
```

**Comment by david-a-wheeler** [comment]
This is an improvement. I suggest we merge something soon, and then we can continue to refine it further as later PRs. Making it readable at all on small displays seems important to me.

**Review by david-a-wheeler — APPROVED** [review]

**Comment by funnelfiasco** [comment]
It made me very sad to remove my emdashes, but I did.

**Review by eddie-knight — APPROVED** [review]
Thanks for doing this @funnelfiasco!

**Pull request merged**